src/data/matrix/basic.lean

```diff
@@ -644,6 +644,14 @@ lemma smul_mul_vec_assoc (A : matrix m n α) (b : n → α) (a : α) :
   (a • A).mul_vec b = a • (A.mul_vec b) :=
 by { ext, apply smul_dot_product }
 
+lemma mul_vec_add (A : matrix m n α) (x y : n → α) :
+  A.mul_vec (x + y) = A.mul_vec x + A.mul_vec y :=
```

Comment:
Do you think you could add this lemma too, which is also missing?

```lean
lemma mul_vec_add (A B : matrix m n α) (x : n → α) :
  (A + B).mul_vec x = A.mul_vec x + B.mul_vec x :=
```
the proof might be the same!